internal/core/items/flickable.rs

```diff
@@ -295,7 +297,9 @@ impl FlickableData {
                 (Flickable::FIELD_OFFSETS.viewport + Rectangle::FIELD_OFFSETS.y)
                     .apply_pin(flick)
                     .set(new_pos.y);
-                InputEventResult::EventAccepted
+                // Although the event was handled, the mouse grabber stack
+                // should extend into child items and don't end at this point
+                InputEventResult::EventIgnored
```

Comment:
I'm a bit concerned about this, why do we want to ignore here?
I think this will mean that the parents are going to handle the event. but we don't really want that, do we?

Reply:
That's what I've tried to explain here: https://github.com/slint-ui/slint/pull/1263#issuecomment-1126882192
I agree. We should get rid of this but just accepting the event breaks everything so another solution has to be found.